gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -488,6 +494,47 @@ gst_tensor_trainer_check_invalid_param (GstTensorTrainer * trainer)
   return TRUE;
 }
 
+/**
+ * @brief Dummy data generation thread
+ */
+static gpointer
+gst_tensor_trainer_dummy_data_generation_func (GstTensorTrainer * trainer)
+{
+  guint i;
+  gint ret = -1;
+  gpointer dummy_data[NNS_TENSOR_SIZE_LIMIT] = { NULL };
+  g_return_val_if_fail (trainer != NULL, NULL);
+
+  gst_tensor_trainer_stop_model_training (trainer);
+
+  do {
+    GST_INFO_OBJECT (trainer, "cur_epoch_data_cnt=%u",
+        trainer->cur_epoch_data_cnt);
+    GST_INFO_OBJECT (trainer, "num_tensors=%d",
+        trainer->prop.input_meta.num_tensors);
+
+    for (i = 0; i < trainer->output_meta.num_tensors; i++) {
+      dummy_data[i] = g_malloc (trainer->push_tensors[i].size);
+      memset (dummy_data[i], 1, trainer->push_tensors[i].size);
```

Comment:
Q> Is there any reason to set 1?

Reply:
No, I thought it would be better than 0. :)